ma_search/data/shape.py

```diff
@@ -17,52 +17,74 @@
 limitations under the License.
 """
 
-import uuid
+import json
 import logging
+from pathlib import Path
+from uuid import UUID, uuid4
 
 import ma_search
+from ma_search.common import logException
+from shapely.geometry import MultiPolygon, Polygon, mapping, shape
```

Comment:
shapely import should be above ma_search import with a blank lune between. (PEP8)